pkg/controller/buildrun/buildrun_controller.go

```diff
@@ -29,9 +30,10 @@ import (
 )
 
 const (
-	namespace     string = "namespace"
-	name          string = "name"
-	pendingReason string = "Pending"
+	namespace          string = "namespace"
+	name               string = "name"
+	pendingReason      string = "Pending"
+	generatedNameRegex        = "-[a-z0-9]{5,5}$"
```

Comment:
Imo you should compile the regexp here.

Reply:
I prefer to compile inside the function, while I need to deal with 2 returned values, the `const` doesnt look like the right place. Thanks for the suggestion.